.github/workflows/build-test-deploy.yml

```diff
@@ -161,13 +161,20 @@ jobs:
 
       - name: Publish unit test results
         uses: EnricoMi/publish-unit-test-result-action@170bf24d20d201b842d7a52403b73ed297e6645b # v2.18.0
-        if: always()
+        if: ${{ !cancelled() }}
```

Comment:
Ah, ok. Does this really happen? I think we set it to `always()` back in the day because that's what is used in this action's examples: https://github.com/EnricoMi/publish-unit-test-result-action

Reply:
Ah, I see CodeCov uses this in their examples (https://docs.codecov.com/docs/test-analytics) so I guess this is fine. :-)